src/app-check/token-generator.ts

```diff
@@ -83,6 +93,7 @@ export class AppCheckTokenGenerator {
         aud: FIREBASE_APP_CHECK_AUDIENCE,
         exp: iat + ONE_HOUR_IN_SECONDS,
         iat,
+        ...customOptions,
```

Comment:
Just curious. Why is `ttl` separate from `exp`?

Reply:
According to go/fac-configurable-ttls we are not exposing the option to use `exp` at all to keep the interface simple. So in this case a custom `ttl` will override `exp`.